internal/eval/eval.go

```diff
@@ -3553,6 +3553,82 @@ func evalJSONNUMINCRBY(args []string, store *dstore.Store) []byte {
 	return clientio.Encode(resultString, false)
 }
 
+// evalJSONOBJKEYS retrieves the keys of a JSON object stored at path specified.
+// It takes two arguments: the key where the JSON document is stored, and an optional JSON path. 
+// It returns a list of keys from the object at the specified path or an error if the path is invalid.
+func evalJSONOBJKEYS(args []string, store *dstore.Store) []byte {
+	if len(args) < 1 {
+		return diceerrors.NewErrArity("JSON.OBJKEYS")
+	}
+
+	key := args[0]
+	// Default path is root if not specified
+	path := defaultRootPath
+	if len(args) > 1 {
+		path = args[1]
+	}
+
+	// Retrieve the object from the database
+	// TODO: Check if we should throw error here
+	obj := store.Get(key)
+	if obj == nil {
+		return clientio.RespNIL
+	}
```

Comment:
thanks for your contribution. in redis. if the key is not exist. should return an error.
`

      127.0.0.1:6379> JSON.SET doc $ '{"a":[3], "nested": {"a": {"b":2, "c": 1}}}'
      OK
      127.0.0.1:6379> JSON.OBJKEYS docs $..a
      (error) ERR could not perform this operation on a key that doesn't exist

`

Reply:
In this [commit](https://github.com/DiceDB/dice/pull/629/commits/946439cf099bc825c1da2bc6b445b2cc1af2328d), 
1. Error will be thrown for non existent key

Reply:
At the next step post getting value of the key, we check if it is json and throw appropriate error So, its handled already.
<img width="543" alt="image" src="https://github.com/user-attachments/assets/3fddeb6f-ae16-436d-a896-2c46b645b718">